apps/babanews/pages/a/[slug].tsx

```diff
@@ -110,7 +110,7 @@ export const getStaticProps: GetStaticProps = async ({params}) => {
     })
   ])
 
-  if (article.data.article) {
+  if (article.data?.article) {
```

Comment:
Changing this fixes runtime errors during SSG which allows to preview draft articles